plugin/bind/README.md

```diff
@@ -17,13 +17,25 @@ If the given argument is an interface name, and that interface has serveral IP a
 
 ## Syntax
 
+In its basic form, a simple bind uses this syntax:
+
 ~~~ txt
 bind ADDRESS  ...
 ~~~
 
-**ADDRESS** is an IP address to bind to.
-When several addresses are provided a listener will be opened on each of the addresses.
+You can also exclude some addresses with their IP address or interface name in expanded syntax:
+
+~~~
+bind ADDRESS ... {
```

Comment:
should this be ADDRESS|IFACE? and throughout ?

Reply:
I explained it throughly in description part, and referenced this section to there for more details